samples/boards/nrf52/power_mgr/README.rst

```diff
@@ -0,0 +1,78 @@
+.. _nrf52-power-mgr-sample:
+
+Power management demo
+#####################
+
+Overview
+********
+
+This sample demonstrares a power manager app that uses the Zephyr
+power management infrastructure to enter into Low Power state.
+
+This app will cycle through the following power schemes each time idle thread
+calls _sys_soc_suspend() hook function :
+
+1. Low Power State: Low Power State is SOC specific and being in this state is
+   transparent to devices. SOC and devices do not lose context in this Mode.
+   SOC supports the following two Low Power states :
+
+   A. CONSTANT LATENCY LOW POWER MODE : This Low Power State triggers CONSTLAT
+      task on nrf52 SOC. In this mode Low Exit Latency. In this Mode there is Low
```

Comment:
"In this mode Low Exit Latency." seems to be remains of a previous version.

Reply:
Thanks for pointing out, When i modified it while addressing the comments, I made this typo. I will change it,